qualtran/bloqs/chemistry/thc/prepare_test.py

```diff
@@ -81,5 +82,14 @@ def test_prepare_graph():
     assert isinstance(sigma, dict)
 
 
+def test_musical_score():
+    uni = _thc_uni()
+    msd = get_musical_score_data(uni)
+    assert isinstance(msd, MusicalScoreData)
+    prep = _thc_prep()
+    msd = get_musical_score_data(prep)
+    assert isinstance(msd, MusicalScoreData)
```

Comment:
so this used to just literally error out?

Reply:
No, it would just print the the same register name for each wire as the (like mu, mu, mu, mu, ... ). BUT my first fix would then cause this to error out because it didn't handle shape > () bitsize > 1 properly (the prep.decompose() would work)